vllm/attention/layer.py

```diff
@@ -10,6 +10,7 @@
 from vllm.attention import AttentionType
 from vllm.attention.selector import backend_name_to_enum, get_attn_backend
 from vllm.config import CacheConfig, get_current_vllm_config
+from vllm.distributed import get_kv_transfer_group, has_kv_transfer_group
```

Comment:
I think having attention layer module depends on kv_connector is not ideal, can we add some callback hook instead?